client_test.go

```diff
@@ -194,7 +194,7 @@ func TestClient_AggRange(t *testing.T) {
 
 	dataPoints, err := client.AggRange(key, ts1, ts2, CountAggregation, 10)
 	assert.Equal(t, nil, err)
-	assert.Equal(t, 2.0, dataPoints[0].Value)
+	assert.Equal(t, 1.0, dataPoints[0].Value)
```

Comment:
Can't we test the full response? This is confusing.

Reply:
Agree :) changing it